extraConfigSriov.py

```diff
@@ -94,7 +94,17 @@ def run(self, _) -> None:
             rh = host.RemoteHost(ip)
             rh.ssh_connect("core")
             result = rh.run("cat /var/lib/ovnk/iface_default_hint").out.strip()
-            print(f"Found PF Name {result} on node {name}")
+            if result:
+                print(f"Found PF Name {result} on node {name}")
+            else:
+                print(f"Cannot find PF Name on node {name} using hint")
+                interface_list = rh.run("sudo ovs-vsctl list-ifaces br-ex").out.strip().split("\n")
+                result = [ x for x in interface_list if "patch" not in x ]
+                result = result[0]
+                if result:
+                    print(f"Found PF Name {result} on node {name}")
+                else:
+                    print(f"Cannot find PF Name on node {name} using ovs-vsctl")
             if result not in pfNamesAll:
                 pfNamesAll.append(result)
 
```

Comment:
Below, we have a fallback to ens1f0. I think your code depends on that too. Let's add a log message there.

Reply:
Done.